test/classes/Collector.spec.js

```diff
@@ -27,7 +27,7 @@ describe('/classes/Collector', ()=> {
     });
   });
 
-  it('著者を保存できる', (done)=> {
+  it('書籍を保存できる', (done)=> {
```

Comment:
__[jshint]__
*  'arrow function syntax (=>)' is only available in ES6 (use esnext option).